pkg/kubectl/cmd/apply_test.go

```diff
@@ -279,6 +296,85 @@ func TestApplyObject(t *testing.T) {
 	}
 }
 
+// TestApplyTPR uses apply commad with ThirdPartyResources.
+func TestApplyTPR(t *testing.T) {
```

Comment:
The behavior was likely in the server-side aspect of registering/updating a TPR (which this doesn't test), not the basic client CRUD of ThirdPartyResource. I'd rather see the apply tests around the third party resource instance data